.github/workflows/release-drafter.yml

```
@@ -9,6 +9,6 @@ jobs:
   update_release_draft:
     runs-on: ubuntu-latest
     steps:
-      - uses: release-drafter/release-drafter@v5.2.0
+      - uses: release-drafter/release-drafter@v5
```

```suggestion
      - uses: release-drafter/release-drafter@v5.9.0
```

I think we would prefer the fixed version here 🙂